.github/workflows/e2e-full.yml

```diff
@@ -26,6 +27,7 @@ jobs:
         include:
           # This is the oldest K8s version we try to support
           - k8s_version: '1.17'
+          - deploytool: 'helm'
```

Comment:
I would add another with globalnet too (as we have seen it to be problematic sometimes)